src/main/python/BUILD_INSTRUCTIONS.md

```diff
@@ -23,27 +23,34 @@ limitations under the License.
 
 The following steps have to be done for both cases
 
-- Build SystemDS with maven first `mvn package -DskipTests`, with the working directory being `SYSTEMDS_ROOT` (Root directory of SystemDS)
+- Build SystemDS with maven first `mvn package -Pdistribution`, with the working directory being `SYSTEMDS_ROOT` (Root directory of SystemDS)
 - `cd` to this folder (basically `SYSTEMDS_ROOT/src/main/python`
 
-### Building package
+### Building package for release
 
 If we want to build the package for uploading to the repository via `python3 -m twine upload --repository-url [URL] dist/*` (will be automated in the future)
 
+- Install twine with `pip install --upgrade twine`
 - Run `create_python_dist.py`
 
 ```bash
 python3 create_python_dist.py
 ```
 
 - now in the `./dist` directory there will exist the source distribution `systemds-VERSION.tar.gz` and the wheel distribution `systemds-VERSION-py3-none-any.whl`, with `VERSION` being the current version number
-- Finished. We can now upload it with `python3 -m twine upload --repository-url [URL] dist/*`
+
+- Follow the instructions from the [Guide](https://packaging.python.org/tutorials/packaging-projects/)
+    1. Create an API-Token in the account (leave the page open or copy the token, it will only be shown once)
+    2. Execute the command `python3 -m twine upload dist/*`
+        - Optional: `pip install keyrings.alt` if you get `UserWarning: No recommended backend was available.`
+    3. Username is `__token__`
+    4. Password is the created API-Token **with** `pypi-` prefix
```

Comment:
Maybe just link to the docs, Then we have one place instead of two with the information

Reply:
Which ones?

Reply:
https://damslab.github.io/docs/sysdspython/install.html

Reply:
I can do this for the `Building for development` but the other case is actually for releasing the package. We probably anyway want to move this somewhere else (and automate it at some point), but for now should I just rename the file `PUBLISH_INSTRUCTIONS.md`?

Reply:
hmm, i think the name change sounds good.

Reply:
Ok :+1: